man/fido_cbor_info_new.3

```diff
@@ -137,6 +139,12 @@ function returns the maximum message size attribute of
 .Fa ci .
 .Pp
 The
+.Fn fido_cbor_info_maxcredbloblen
+function returns the maximum credBlob length in bytes supported by the
+authenticator, as reported in
```

Comment:
-comma here, or +comma everywhere else in this doc page w/ a similar construction :)

Reply:
fixed in 3e7e37e1ead79f0539b18bafee58039cc618c3ea